Sources/Datadog/RUM/Instrumentation/Views/RUMViewsHandler.swift

```diff
@@ -193,3 +171,56 @@ internal final class SwiftUIRUMViewsHandler: SwiftUIViewHandler {
         }
     }
 }
+
+extension RUMViewsHandler: UIViewControllerHandler {
+    func notify_viewDidAppear(viewController: UIViewController, animated: Bool) {
+        if let view = stack.first(where: { $0.identity.equals(viewController) }) {
+            // If the stack already contains the view controller, just restarts the view.
+            // This prevent from calling the predicate when unnecessary.
+            add(view: view)
+        } else if let rumView = predicate?.rumView(for: viewController) {
+            add(
+                view: .init(
+                    identity: viewController,
```

Comment:
This will keep a strong ref to the `viewController`, even if `notify_viewDidDisappear` will be called to release it from the stack, we should keep a `weak` ref to the vc.
This should be addressed before merging.